docs/framework/svelte.md

```diff
@@ -0,0 +1,128 @@
+
+#### useWallet
+
+The `useWallet` primitive provides access to wallet management features. Here's an example showing some commonly used values:
+
+```sv
+<script lang="ts">
+  import { useWallet } from '@txnlab/use-wallet-svelte'
+  const { 
+    wallets,             // List of available wallets
+    activeWallet,        // Currently active wallet (function)
+    activeAccount,       // Active account in active wallet
+    activeAddress,       // Address of active account
+    isReady,             // Whether all wallet providers have finished initialization
+    signTransactions,    // Function to sign transactions
+    transactionSigner,   // Typed signer for ATC and Algokit Utils
+    algodClient          // Algod client for active network
+  } = useWallet()
+</script>
+
+{#if isReady()}
+  {#if activeAddress.current}
+    <div>Connected: {activeAddress.current}</div>
+  {:else}
+    <div>Not connected</div>
+  {/if}
+{:else}
+  <div>Loading...</div>
+{/if}
+```
+
+Note that unlike [React's hooks](react.md#using-the-hooks), Svelte's primitives return functions that need to be called to access their current value. The values are automatically tracked and will trigger reactivity when they change.
+
+For a complete list of all available properties and methods, see the [useWallet API Reference](../api-reference/usewallet.md).
+
+#### useNetwork
+
+The `useNetwork` primitive serves two primary functions: managing the active network and supporting runtime node configuration.
+
+```sv
+<script lang="ts">
+  import { useNetwork } from '@txnlab/use-wallet-svelte'
+
+  const {
+    // Active network management
+    activeNetwork,         // Currently active network
+    setActiveNetwork,      // Function to change networks
+    
+    // Runtime node configuration
+    networkConfig,         // Complete configuration for all networks
+    activeNetworkConfig,   // Configuration for active network only
+    updateAlgodConfig,     // Update a network's Algod configuration
+    resetNetworkConfig     // Reset network config to initial values
+  } = useNetwork()
+</script>
+
+<div>
+  <!-- Example: Network selector dropdown -->
+  <select
+    value={activeNetwork.current}
+    onchange={(e) => setActiveNetwork(e.currentTarget.value)}
+  >
+    {#each Object.keys(networkConfig()) as networkId}
+      <option>
+        {networkId}
+      </option>
+    {/each}
+  </select>
+</div>
+```
+
+Active network management (previously part of `useWallet`) enables users to switch between different networks.
+
+Runtime node configuration, introduced in v4.0.0, enables users to override the application's default node settings and connect to any Algorand node. See the [Runtime Node Configuration](../guides/runtime-node-configuration.md) guide for details about implementing this feature.
+
+For a complete list of all available properties and methods, see the [useNetwork API Reference](../api-reference/usenetwork.md).
+
```

Comment:
Before "Next Steps" why don't we add a "Design Decisions" section that calls out the `.current` reactivity pattern and explains the trade-offs. Based on Kappa's initial reaction, I get the impression that this should probably be explained.

While a Svelte developer might question why you didn't use runes, you can explain that it was a pragmatic decision for consistency with the other framework adapters (by making it compatible with Tanstack Store).